package.json

```diff
@@ -2,9 +2,9 @@
   "name": "status-bot",
   "version": "3.0.0",
   "description": "",
-  "main": "index.js",
+  "main": "src/index.js",
```

Comment:
`index.js` is the correct one here. 

`./src/index.js` - is the main class. 
`index.js` - is the class users are supposed to fill out and run.